doc/en/user/source/services/wms/reference.rst

```diff
@@ -119,6 +119,10 @@ They are fully documented in the :ref:`wms_vendor_parameters` section.
    * - ``format``
      - No
      - request the capabilities document in a certain format
+   * - ``rootLayer``
+     - No
+     - Flag to enable/disable the standard Root top level Layer element. Values are true or false. When false, the Root element will be included only if there is NOT a single top level element already.
```

Comment:
```suggestion
     - Flag to enable/disable the standard Root top level Layer element. Values are true or false. When false, the Root element will be included only if there are multiple top level layers, if there is only one, it will be the root layer itself.
```